pylintrc

```diff
@@ -37,7 +37,7 @@ load-plugins=
 # W0141 = Used builtin function 'filter'
 # W0201 = Attribute defined outside __init__
 # R0201 = Method could be a function
-disable=W0141,W0201,R0201,too-few-public-methods,invalid-name,missing-docstring,superfluous-parens,duplicate-code
+disable=W0141,W0201,R0201,too-few-public-methods,invalid-name,missing-docstring,superfluous-parens,duplicate-code,bare-except,broad-except
```

Comment:
bare except are nearly always wrong. I would like to not disable this warning in general. I would rather suggest to pylint-ignore all instances we have right now, maybe with some #fixme comment. But any new occurrence should produce warning.

Reply:
absolutely agree (thats what I mentioned in the commit message). But I agree that it would be better to disable each one separately, so it notifies about a new ones. Will rework it.